build.rs

```diff
@@ -41,6 +41,19 @@ mod musl_reference_tests {
         "rem_pio2.rs",
         "rem_pio2_large.rs",
         "rem_pio2f.rs",
+        "remquo.rs",
+        "remquof.rs",
+        "lgamma.rs",  // lgamma passed, lgamma_r has more than 1 result
+        "lgammaf.rs", // lgammaf passed, lgammaf_r has more than 1 result
+        "frexp.rs",   // more than 1 result
+        "frexpf.rs",  // more than 1 result
+        "sincos.rs",  // more than 1 result
+        "sincosf.rs", // more than 1 result
+        "modf.rs",    // more than 1 result
+        "modff.rs",   // more than 1 result
+        "asinef.rs",  // not exists
+        "jn.rs",      // passed, but very slow
+        "jnf.rs",     // passed, but very slow
```

Comment:
Can you comment on the slowness? How so? Could the number of tests simply be reduced?

(I would prefer to ignore literally nothing that is a public interface of this library)

Reply:
About 10 minutes with NTESTS = 5

```
test ynf_matches_musl ... ok
test jn_matches_musl ... test jn_matches_musl has been running for over 60 seconds
test jnf_matches_musl ... test jnf_matches_musl has been running for over 60 seconds
test jn_matches_musl ... ok
test jnf_matches_musl ... ok
```

Reply:
With only 5 tests it takes 5 minutes to run these tests? Does that mean these are just too slow to land?

Reply:
`jn(n: i32, mut x: f64) -> f64` is Bessel function where `n` is order.
I `n` in think tests for `jn` and `jnk` should be limited to small values like 0, 1, .. 5.